files/tg4810_dts.patch

```diff
@@ -0,0 +1,1405 @@
++++            };
++++            sys-amber{
++++                    gpios = <&cp0_gpio1 31 GPIO_ACTIVE_LOW>;
++++                    default-status = "off";
++++                    label = "sysLedAmber";
++++            };
++++        };
++++};
++++
++++AP_UART: &uart0 {
++++	status = "okay";
++++};
++++
++++AP_I2C: &i2c0 {
++++	status = "okay";
++++	clock-frequency = <100000>;
++++};
++++
++++&cp0_pinctrl {
++++        cp0_ge_mdio_pins: ge-mdio-pins {
++++                marvell,pins = "mpp0", "mpp1";
++++                marvell,function = "ge";
++++        };
++++        cp0_sata0_prsnt_pin: sata0-prsnt-pins {
++++                marvell,pins = "mpp29";
++++                marvell,function = "sata0";
++++        };
++++        cp0_i2c0_pins: i2c0-pins {
++++                marvell,pins = "mpp37", "mpp38";
++++                marvell,function = "i2c0";
++++        };
++++        cp0_i2c1_pins: i2c1-pins {
++++                marvell,pins = "mpp35", "mpp36";
++++                marvell,function = "i2c1";
++++        };
++++        cp0_spi0_pins: spi0-pins {
++++                marvell,pins = "mpp56", "mpp57", "mpp58", "mpp59";
++++                marvell,function = "spi0";
++++        };
++++};
++++
++++&cp0_spi0 {
++++	status = "okay";
++++	pinctrl-names = "default";
++++	pinctrl-0 = <&cp0_spi0_pins>;
++++	spi-flash@0 {
++++        	#address-cells = <0x1>;
++++                #size-cells = <0x1>;
++++                compatible = "jedec,spi-nor";
++++                reg = <0x0>;
++++                spi-max-frequency = <0x1312d00>;
++++
++++              	partitions {
++++            		compatible = "fixed-partitions";
++++                  	#address-cells = <0x1>;
++++                  	#size-cells = <0x1>;
++++
++++                 	partition@0 {
++++                    		label = "uboot";
++++                    		reg = <0x0 0x3f0000>;
++++        		};
++++               		partition@3f0000 {
++++                    		reg = <0x3f0000 0x10000>;
++++                  		label = "uboot-env";
++++             			env_size = <0x10000>;
++++        		};
++++      			partition@400000 {
++++           			reg = <0x400000 0xc00000>;
++++                 		label = "onie";
++++              		};
++++   		};
++++	};
++++};
++++
++++&cp0_i2c0 {
++++	status = "okay";
++++	clock-frequency = <100000>;
++++	pinctrl-names = "default";
++++	pinctrl-0 = <&cp0_i2c0_pins>;
++++
++++        adt7473@2e {
++++            compatible = "adi,adt7473";
++++            reg = <0x2e>;
++++        };
++++        lm75a@4a {
++++            compatible = "ti,tmp75";
++++            reg = <0x4a>;
++++        };
++++        lm75a@4b {
++++            compatible = "ti,tmp75";
++++            reg = <0x4b>;
++++        };
++++        i2c-mux@70 {
++++            compatible = "nxp,pca9546";
++++            reg = <0x70>;
++++            #address-cells = <1>;
++++            #size-cells = <0>;
++++            i2c@0 {
++++                #address-cells = <1>;
```

Comment:
this one is for fan eeprom 0x54,0x55, 0x56, and -x40 for which  dont add any sub i2c devices.
Just add ony one and change reg value to 0xFF